libkbfs/tlf_journal.go

```diff
@@ -864,7 +869,6 @@ func (j *tlfJournal) flush(ctx context.Context) (err error) {
 			if j.singleOpMode == singleOpFinished {
 				j.log.CDebugf(ctx, "Resetting single op mode")
 				j.singleOpMode = singleOpRunning
-				j.singleOpLockContext = nil
```

Comment:
I don't remember why I had `singleOpLockContext` as part of the `tlfJournal` struct, so I moved it into this function here. If anybody knows my earlier thought process better than me, please let me know!

Reply:
Ugh I think I remember now. NVM ...